ACPs/23-p-chain-native-transfers.md

```diff
@@ -0,0 +1,58 @@
+```text
+ACP: <PR Number>
+Title: P-chain native transfers
+Author(s): Dhruba Basu <https://github.com/dhrubabasu>
+Discussions-To: <GitHub Discussion URL>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Support native transfers on P-chain.
+
+## Motivation
+
+Currently, the P-chain has no simple transfer transaction type. The X-chain supports this functionality through a `BaseTx`. Although the P-chain contains transaction types that extend `BaseTx`, the `BaseTx` transaction type itself is not a valid transaction.
```

Comment:
Can you elaborate on advantages of `BaseTx` being made a valid transaction, vs using one of the current transaction types that extend `BaseTx`?

Reply:
Elaborated 👍🏾